freqtrade/freqai/freqai_interface.py

```diff
@@ -301,18 +301,41 @@ def start_backtesting(
             dk.set_new_model_names(pair, timestamp_model_id)
 
             if dk.check_if_backtest_prediction_is_valid(len_backtest_df):
-                self.dd.load_metadata(dk)
-                dk.find_features(dataframe)
-                self.check_if_feature_list_matches_strategy(dk)
+                # self.dd.load_metadata(dk)
```

Comment:
@robcaulk Can we remove this features check? I modified the logic and we don't have the features anymore if it's using backtesting saved predictions.